app/models/scene_shop.rb

```diff
@@ -3,4 +3,12 @@ class SceneShop < ApplicationRecord
   has_many :emails, as: :recipient, dependent: :destroy
   has_many :phones, as: :owner, dependent: :destroy
   has_many :addresses, as: :location, dependent: :destroy
+
+  def address
```

Comment:
Now that I see it.. this is duplicated over every model, but why?

Does it make more sense for this to live in the Address class? so you can call .to_s on the Address and it'll print itself prettily?